cartography/intel/aws/__init__.py

```diff
@@ -144,21 +145,35 @@ def _sync_multiple_accounts(
     logger.info("Syncing AWS accounts: %s", ', '.join(accounts.values()))
     organizations.sync(neo4j_session, accounts, sync_tag, common_job_parameters)
 
+    failed_account_ids = []
+    exception_tracebacks = []
+
     for profile_name, account_id in accounts.items():
         logger.info("Syncing AWS account with ID '%s' using configured profile '%s'.", account_id, profile_name)
         common_job_parameters["AWS_ID"] = account_id
         boto3_session = boto3.Session(profile_name=profile_name)
 
         _autodiscover_accounts(neo4j_session, boto3_session, account_id, sync_tag, common_job_parameters)
 
-        _sync_one_account(
-            neo4j_session,
-            boto3_session,
-            account_id,
-            sync_tag,
-            common_job_parameters,
-            aws_requested_syncs=aws_requested_syncs,  # Could be replaced later with per-account requested syncs
-        )
+        try:
+            _sync_one_account(
+                neo4j_session,
+                boto3_session,
+                account_id,
+                sync_tag,
+                common_job_parameters,
+                aws_requested_syncs=aws_requested_syncs,  # Could be replaced later with per-account requested syncs
+            )
+        except Exception as e:
```

Comment:
To help with debugging, it's also helpful to do a `logger.exception(...)` here so that the timestamp of the exception can be recorded.

Reply:
Just talked with @ramonpetgrave64 and this is what we should do:

- let's include the failing account ID and the timestamp next to each traceback.
- there is no need to do logger.exception on line 168.

The main idea is to collect all the exceptions as they happen and wait until the end before logging at `error` level. We only want to log at `error` level right before when the program is fatally failing.